### PR TITLE
Add `bundleType` to quote

### PR DESCRIPTION
**Pull request by robinandeer**
# Jira Issue: []

## What?

Make it possible to differentiate main, addon, standalone quotes.
Move out inner function (`priority`).

## Why?

To support our "bundling" possibilities on the offer page.

## Optional checklist
- [ ] Unit tests written
- [ ] Tested locally
- [ ] Codescouted

### CODE REVIEW

File: src/resolvers/quote-bundle/quoteBundle.graphqls [hunk 1]
```diff
@@ -13,6 +13,12 @@ type QuoteBundle {
   displayName(locale: Locale!): String!
 }
 
+enum BundledQuoteType {
```
Comment (hugokallstrom) [inline src/resolvers/quote-bundle/quoteBundle.graphqls:16]: Could we add some comments here to make it clear what the difference is between eg. main and standalone?

**Review by hugokallstrom — APPROVED** [review]

File: src/resolvers/quote-bundle/quoteBundle.graphqls (hunk 1 above)
Comment (robinandeer) [reply src/resolvers/quote-bundle/quoteBundle.graphqls:16]: Ahh yes that's a very good point!

File: src/resolvers/quote-bundle/quoteBundle.graphqls [hunk 2]
```diff
@@ -31,6 +37,7 @@ type BundledQuote {
   typeOfContract: TypeOfContract!
   initiatedFrom: String!
   displayName(locale: Locale!): String!
+  bundleType: BundledQuoteType!
```
Comment (guilhermespopolin) [inline src/resolvers/quote-bundle/quoteBundle.graphqls:40]: Nit:
Wouldn't something like `type` or `category` be a better name? `BundledQuote` it's about one specific quote and `bundleType` sounds like the type of a bundle.

**Review by guilhermespopolin — APPROVED** [review]

File: src/resolvers/quote-bundle/quoteBundle.graphqls (hunk 2 above)
Comment (robinandeer) [reply src/resolvers/quote-bundle/quoteBundle.graphqls:40]: Sure, I also don't like the name but it's the best I had.

We already have a "type" field for the "insurance type" so don't want to use that.

Category also feels a bit too generic, no?

Comment (guilhermespopolin) [reply src/resolvers/quote-bundle/quoteBundle.graphqls:52]: Yeah, naming is hard 😞

**Pull request closed without merging**